qemu/tests/seabios_bin.py

```diff
@@ -0,0 +1,58 @@
+import logging
+
+from virttest import error_context
+from virttest import env_process
+from virttest import utils_misc
+
+from avocado.utils import process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    KVM Seabios bin file test:
+    1) Get available bin files
+    2) Get supported machine types
+    3) Check bin file in use
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    bin_dict = {'rhel6': 'bios.bin', 'rhel7': 'bios-256k.bin'}
```

Comment:
how about rhel8? same as rhel7? any need to add a pair of key and value?

Reply:
The supported machine type in rhel8 is not concluded, so no need to consider rhel8 for now. Thanks!